src/conv_bias_act.jl

```diff
@@ -0,0 +1,34 @@
+export conv_bias_act, conv_bias_act!
+
+function conv_bias_act(x::AbstractArray{xT,N}, w::AbstractArray{wT,N}, cdims::ConvDims, b::AbstractArray{bT,N}, σ=identity; kwargs...) where {xT, wT, bT, N}
```

Comment:
the bias here doesn't have dimension N

Reply:
never mind the previous comment: in Flux's Conv layers, the bias (a vector) is reshaped to the same dimension of x and w when applied